addons/css-fix/userstyle.css

```diff
@@ -0,0 +1,18 @@
+
+.blocklyWidgetDiv .goog-menu {
+overflow: hidden; /* Hide scrollbars */
+}
+.sprite-selector_items-wrapper_4bcOj.box_box_2jjDp {
+overflow: hidden;
+}
+.sprite-selector_items-wrapper_17MEf {
+overflow: hidden; /* Hide scrollbars */
+}
+.s3devSTT {
+border-top: 1px solid hsla(0, 0%, 0%, 0.15);
+padding: 8px 12px;
+}
+.react-contextmenu-item:hover {
+background: hsla(215, 100%, 65%, 1);
+color: white;
+}
```

Comment:
What is this for?

Reply:
I have removed that, it's not supposed to be there